lib/killbill_client/models/invoice_item.rb

```diff
@@ -82,6 +82,18 @@ def delete(user = nil, reason = nil, comment = nil, options = {})
                               :comment => comment,
                           }.merge(options)
       end
+
+      def create_tax_items(auto_commit = false, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
Maybe `create_tax_item`, since a single item is created.

Reply:
@pierre I pushed the requested change.